pkg/util/util.go

```diff
@@ -177,3 +177,36 @@ func GetKubeClient(kubeconfig string) (*kubernetes.Clientset, error) {
 
 	return kubernetes.NewForConfig(config)
 }
+
+// Function copied from kubernetes/pkg/util/slice/slice.go
+func ContainsString(slice []string, s string, modifier func(s string) string) bool {
```

Comment:
curious as to why these need to be copied in rather than using the slice implementations as is?

Reply:
importing from kube core is not recommended. i suspect this would block upstreaming. we could try to pr moving the slice functions into one of the libs kube provides to be imported, but for now i just copied.

Reply:
i'll avoid a rant.  but if it's not currently published as a library and we shouldn't abuse the core impl just because it's visible, copy-pasta it is.